src/kategorier/kategorier.ts

```diff
@@ -10,10 +12,12 @@ export interface InngangKategori {
 export interface Kategori {
     temaKey: TemaKey;
     title: string;
-    anke: boolean;
+    allowsAnke: boolean;
     digitalKlage: boolean;
     digitalAnke: boolean;
```

Comment:
`digitalAnke` blir aldri brukt etter hva jeg kan se. Er den for fremtidig bruk?

Reply:
> `digitalAnke` blir aldri brukt etter hva jeg kan se. Er den for fremtidig bruk?

Den brukes i `routes.tsx`:

``` if (digitalKlage) {
        return (
            <InngangInnsendingDigital/> ...```

Reply:
`digitalKlage` blir brukt i `routes.tsx`, ja, men det er `digitalAnke` jeg ikke klarer finne noe sted :)

Reply:
> `digitalKlage` blir brukt i `routes.tsx`, ja, men det er `digitalAnke` jeg ikke klarer finne noe sted :)

Sant det ja. :) Den brukes ikke, men ble nevnt i kategorilisten - men mulig vi kan legge den inn når det faktisk er relevant ja.